portality/app.py

```diff
@@ -113,6 +113,14 @@
 SPONSORS = {k: OrderedDict(sorted(v.items(), key=lambda t: t[0])) for k, v in SPONSORS.items()}
 
 
+# Configure the Google Analytics tracker
+from portality.lib import analytics
+try:
+    analytics.create_tracker(app.config['GOOGLE_ANALYTICS_ID'], app.config['BASE_DOMAIN'])
+except KeyError:
+    app.logger.warn("No Google Analytics credentials found. Required: 'GOOGLE_ANALYTICS_ID' and 'BASE_DOMAIN'.")
```

Comment:
This feels too important in production for a mere warning. Is this because there's no sensible values to set these to in development?

Reply:
Well GA is really entirely optional, so I didn't think it was an error to omit the details. Do you suggest I make it an error with a check for env=production? That'd be even less bothersome for developers I suppose.